docs/overview.md

```diff
@@ -24,22 +30,27 @@ Users may use the Stacks.js CLI directly to send a transaction.
 Backends may need to automate signing without direct user interaction.
 In these cases, developers can use the same libraries used by Stacks wallets for account handling and transaction signing.
 
----
+-->
+--- 
 
-There are three main integrations used by Stacks enabled applications:
+While there are many Stacks.js use cases, we find that there are three main integrations used by Stacks-enabled applications:
 
 <!-- todo: add a card and better how-to-guide for each, (eg add message signing backend checking) -->
 
 - **Authentication**: Register and sign users in with identities on the Stacks blockchain
 - **Transaction signing**: Prompt users to sign and broadcast transactions to the Stacks blockchain
 - **Data storage**: Save and retrieve data for users with [Gaia](https://docs.stacks.co/build-apps/references/gaia)
```

Comment:
i would almost remove the callout to Gaia here

Reply:
and maybe we add reading state from a smart contract (calling smart contract functions)

Reply:
Agreed. Updated with https://github.com/hirosystems/stacks.js/pull/1568/commits/f550d16d0169666d34d8555385e140c732c7d08e